includes/class-convertkit-output.php

```diff
@@ -338,6 +338,35 @@ public function append_form_to_content( $content ) {
 				$content = $form . $content;
 				break;
 
+			case 'after_element':
+				$form_position_element            = $this->settings->get_default_form_position_element( get_post_type( $post_id ) );
+				$form_position_element_index      = $this->settings->get_default_form_position_element_index( get_post_type( $post_id ) );
+				$form_position_element_tag_length = ( strlen( $form_position_element ) + 3 );
+
+				// Find all closing elements.
+				preg_match_all( '/<\/' . $form_position_element . '>/', $content, $matches );
```

Comment:
Some elements don't have end tags, e.g., `<img />`. What would you think of parsing with a real HTML parser? Requisite [StackOverflow](https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags)